schemas/session-replay-mobile-events-schema.json

```diff
@@ -6,10 +6,16 @@
   "description": "Schema of all properties of a mobile Session Replay event",
   "oneOf": [
     {
-      "$ref": "session-replay/shape-wireframe-schema.json"
+      "$ref": "session-replay/mobile/shape-wireframe-schema.json"
     },
     {
-      "$ref": "session-replay/text-wireframe-schema.json"
+      "$ref": "session-replay/mobile/text-wireframe-schema.json"
+    },
+    {
+      "$ref": "session-replay/segment-schema.json"
+    },
+    {
+      "$ref": "session-replay/record-schema.json"
```

Comment:
**question:** why add the subtypes and not just the segment that includes all of them?

Reply:
good point